### ISSUE TITLE
[TalkBack] Delay between mounting SD card and loading TTS causes failover

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#121]
```
What steps will reproduce the problem?
1. Switch to either Ivona or Svox classic
2. When talkback switches the voice, turn off the device
3. Turn it back on

What is the expected output? What do you see instead?
Talkback should come up speaking. Instead, all you hear is sounds,and you need 
either sighted help or another screen reader to change voices.

What version of the product are you using? On what operating system?
Tested talkback 3.08 through 1.2.3, xPeria pro with ICS 4.04.

Please provide any additional information below.

This seems to only happen with larger voices. Pico, espeak and google TTS work 
fine.
```

Original issue reported on code.google.com by `piterm...@gmail.com` on 18 Aug 2012 at 9:02

**Comment by GoogleCodeExporter** [comment]
```
I'm experiencing something similar.  If I do a fresh install of SVOX and set it 
to the default TTS engine, then start Talkback, things are working.  But then 
if I turn off Talkback and turn it back on, I get nothing - no speech.  The 
only way I've found to fix it is to uninstall/reinstall SVOX.  I did not have 
this problem until my Droid 4 was recently upgraded to 4.0.4.
```

Original comment by `steve...@gmail.com` on 21 Aug 2012 at 5:50

**Comment by GoogleCodeExporter** [comment]
```
fyi.... The work-around for this is to stop Talkback by going to Settings -> 
Apps -> Talkback and do a "Force stop".  Then when you restart Talkback, SVOX 
works.
```

Original comment by `steve...@gmail.com` on 23 Aug 2012 at 12:29

**Comment by GoogleCodeExporter** [comment]
Original comment by `caseybur...@google.com` on 13 Sep 2012 at 6:13
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
TalkBack attempts to load the user-specified default TTS engine when the SD 
card is mounted, but if the engine takes a long time to load then it will fail 
again and become blacklisted.
```

Original comment by `alanv@google.com` on 7 Dec 2012 at 6:45
- Changed title: **[TalkBack] Delay between mounting SD card and loading TTS causes failover**

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in the current Play Store version of TalkBack. If issue 
persists, please reopen this bug.
```

Original comment by `alanv@google.com` on 30 Apr 2013 at 8:02

**Comment by GoogleCodeExporter** [comment]
Original comment by `alanv@google.com` on 30 Apr 2013 at 8:02
- Changed state: **Fixed**
